src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -85,31 +86,53 @@ const PendingApplications = () => {
   const [applications, setApplications] = useState([]);
   const [modalIsVisible, setModalIsVisible] = useState(false);
   const [profileId, setProfileId] = useState('');
-  const { axiosWithAuth } = useAxiosWithAuth0();
 
   const showModal = profile_id => {
     setProfileId(profile_id);
     setModalIsVisible(true);
   };
 
+  const convertDate = previousDate => {
+    const timestamp = new Date(previousDate);
+    const newConvertedDate = timestamp.toLocaleString();
+    if (newConvertedDate === 'Invalid Date') {
+      return '';
+    }
+    return newConvertedDate;
+  };
+
   const getPendingApps = async () => {
     try {
-      const api = await axiosWithAuth().post(`/application`);
-      api.data.forEach(row => {
+      const api = await axios.get('/local.json');
+      api.data.result.forEach(row => {
         row.hasOwnProperty('accepting_new_mentees')
           ? (row.role_name = 'mentor')
           : (row.role_name = 'mentee');
       });
       setApplications(
-        Object.values(api.data).map(row => ({
+        Object.values(api.data.result).map(row => ({
           key: row.profile_id,
-          name: row.first_name + ' ' + row.last_name,
+          first_name: row.first_name,
+          last_name: row.last_name,
+          tech_stack: row.tech_stack,
+          role_name: row.role_name,
+          email: row.email,
+          state: row.state,
+          country: row.country,
+          current_position: row.current_position,
+          current_company: row.current_company,
+          industry_knowledge: row.industry_knowledge,
+          pair_programming: row.pair_programming,
+          job_help: row.job_help,
+          industry_knowledge: row.industry_knowledge,
+          other_info: row.other_info,
+          full_name: row.first_name + ' ' + row.last_name,
           role: (
             <Tag color={row.role_name === 'mentor' ? 'blue' : 'purple'}>
               {row.role_name}
             </Tag>
           ),
-          date: (row.updated_at ? row.updated_at : row.created_at).slice(0, 10),
+          date: convertDate(row.updated_at),
           status: (
```

Comment:
working with dates can be tricky, great Job converting it to a user friendly readable format